### PR TITLE
Updated simple example in README.md

### LINKED ISSUE #884: Update README cpp example to reflect latest API changes.

### PR DESCRIPTION
**Pull request by tkuchida**
Added ConsoleReporter and made other small improvements (e.g., more meaningful names). Changed from arm to leg because "humerus" was too long for line 56. Added console output and updated code in README.md.

Fixes #884 
@chrisdembia @jimmyDunne

### CODE REVIEW

File: OpenSim/Tests/README/testREADME.cpp [hunk 1]
```diff
@@ -43,48 +44,57 @@
...
     model.addForce(muscle);
     model.addController(brain);
 
+    // Add a console reporter to print the muscle fiber force and knee angle.
+    ConsoleReporter* reporter = new ConsoleReporter();
+    reporter->set_report_time_interval(1.0);
+    reporter->updInput("inputs").connect(muscle->getOutput("fiber_force"));
+    reporter->updInput("inputs").connect(
+        knee->getCoordinateSet()[0].getOutput("value"));
+    model.addComponent(reporter);
```
Comment (tkuchida) [inline OpenSim/Tests/README/testREADME.cpp]: `addComponent()` should come before `connect()`.

**Comment by chrisdembia** [comment]
LGTM except I would prefer to leave it as an arm example. The gif definitely looks like an arm more than a leg, and I don't think we should go through the hassle of updating the gif. Also, the muscle is still called "biceps" (although I suppose this could be the BFSH/LH).  Also, the README displays as more than 80 columns on github, so I am not too concerned about column width here.

**Comment by tkuchida** [comment]
> LGTM except I would prefer to leave it as an arm example.

Ok, I will change it back and violate our column width rule.

**Comment by chrisdembia** [comment]
LGTM please merge when tests pass.

**Pull request merged**